server/helpers/dataProcessor.js

```diff
@@ -26,4 +27,17 @@ const processTimeToMins = function (obj, dataObj, dataArr) {
   dataArr.push(dataObj);
 };
 
-module.exports = { compileData };
+// Helper function to remove 'www.' and '.com' from URLs to determine URL's name
+const extractNameFromURL = function (url) {
+  const remPrefix = url.split("www.").join("");
+  const remSuffix = remPrefix.split(".")[0];
+  const name = remSuffix.charAt(0).toUpperCase() + remSuffix.slice(1);
+  return name;
+};
+
+// Helper function to remove "www." if a host_name has it
+const remPrefix = function (host_name) {
+  return host_name.split("www.").join("");
+};
+
+module.exports = { compileData, extractNameFromURL, remPrefix };
```

Comment:
Minor thing but if you don't plan to call remPrefix on it's own, you don't actually need to export it here and can just export extractNameFromURL